src/main/resources/application.properties

```diff
@@ -9,7 +9,7 @@
 #spring.datasource.username=postgres
 #spring.datasource.password=postgres
 #spring.jpa.generate-ddl=true
-server.port=8080
+server.port=8081
```

Comment:
Changes to the port should not be committed, otherwise automated deploy will fail.

Also, have the docker/*, .mvn/* and mvnw.cmd files been deleted? They should be present in the repository, I believe.

Reply:
He created a new pull request for this. 

The new one: https://github.com/fogbow/authentication-service/pull/70